include/highfive/h5easy_bits/H5Easy_scalar.hpp

```diff
@@ -94,13 +96,8 @@ struct dump_extend_impl
         file.flush();
         return dataset;
     }
-};
-
-// load "scalar" as part of a larger DataSet
-template <class T>
-struct load_impl
-{
-    static T run(const File& file,
+    // load a part of a larger DataSet
```

Comment:
The previous comment "a scalar" still remains true no? That being a said, a static_assert could be a nice extension here. 

Furthermore, it would be good to keep the white-line between functions.